ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -103,18 +103,12 @@ fn deserialise_into_ti_vec<I, T>(v: Vec<T>) -> Result<TiVec<I, T>, DekuError> {
 
 /// A trait for converting in-memory data-structures into a human-readable textual format.
 ///
-/// This is modelled on [`std::fmt::Display`], but a reference to the module is always passed down
-/// so that constructs that require lookups into the module's tables from stringification have
-/// access to them.
-///
-/// The way we implement this (returning a `String`) is inefficient, but it doesn't hugely matter,
-/// as the human-readable format is only provided as a debugging aid.
+/// This is analogous [std::fmt::Debug], but:
```

Comment:
True, one could argue this either way.

Reply:
Sorry to be a pain, but since the output is intended for human consumption, and the trait is called `AotIRDisplay` (i.e. contains the word "Display", because it's modelled on `Display`), I think it'd be better to explain in terms of `Display`